storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -573,10 +573,6 @@ public static ProxyConfig getProxyConfig() {
         return new ProxyConfig(type, host, port, username, password);
     }
 
-    public static boolean isQueueLocked() {
-        return prefs.getBoolean(PREF_QUEUE_LOCKED, false);
```

Comment:
Now `PREF_QUEUE_LOCKED` is probably unused, could you please remove that as well?